ledger/participant-integration-api/src/main/scala/platform/store/cache/MutableCacheBackedContractStore.scala

```diff
@@ -296,6 +297,25 @@ object MutableCacheBackedContractStore {
       ContractsStateCache(maxContractsCacheSize, metrics),
     )
 
+  def owner(
+      contractsReader: LedgerDaoContractsReader,
+      signalNewLedgerHead: Offset => Unit,
+      metrics: Metrics,
+      maxContractsCacheSize: Long = 100000L,
```

Comment:
Is it necessary to define the defaults here, considering they are defined elsewhere as well?

Reply:
You're right, they're redundant. Bubbled them up in the CLI config